Makefile

```diff
@@ -27,35 +27,28 @@ endif
 # Configuration - Tooling
 # ------------------------------------------------------------------------------
 
-CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
+PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+
+.PHONY: _download_tool
+_download_tool:
+	(cd tools && go mod tidy ) && \
+		GOBIN=$(PROJECT_DIR)/bin go install -modfile tools/go.mod $(TOOL)
```

Comment:
:+1:

Reply:
Dapr imports external tools https://github.com/dapr/dapr/blob/master/tools/tools.go

Reply:
If you feel strongly about using `tools/` I could be persuaded, but if you're on board for `third_party` I'm just more used to seeing repositories that use `tools/` for tools where the source is maintained therein.